### PR TITLE
Join flow country placeholder

### PR DESCRIPTION
**Pull request by benjiwheeler**
### Resolves:

Step towards resolving https://github.com/LLK/scratch-www/issues/3053

### Changes:

* changes placeholder for country to "Select country"

### CODE REVIEW

File: src/l10n.json [hunk 1]
```diff
@@ -18,7 +18,6 @@
     "general.conferences": "Conferences",
     "general.copyright": "Scratch is a project of the Lifelong Kindergarten Group at the MIT Media Lab",
     "general.copyrightDraft": "Scratch is a project of the Scratch Foundation in collaboration with the Lifelong Kindergarten Group at the MIT Media Lab.",
-    "general.country": "Country",
```
Comment (picklesrus) [inline src/l10n.json]: Is this still used here? https://github.com/LLK/scratch-www/blob/46de5a23e17c5d3a5226e29e54f09174c3264706/src/components/registration/steps.jsx#L556

Comment (benjiwheeler) [reply src/l10n.json]: You're right! I put it back.

**Comment by apple502j** [comment]
Question BTW: Why do you call it "country", when it includes non-countries?

**Comment by benjiwheeler** [comment]
> Question BTW: Why do you call it "country", when it includes non-countries?

That seemed like the clearest name for the category, which uses the [ISO list of country names and codes](https://www.iso.org/iso-3166-country-codes.html). Do you have another suggestion?

**Pull request merged**